### ISSUE TITLE
Add a dark theme

### ISSUE THREAD

**Issue by VFDan** [MasterOfTheTiger/reimagine-scratch#2]
A dark theme toggle would help with people using Scratch at night. 
Here is an example: https://vimeo.com/327395700 (Password is VFDan)

**Comment by moeenio** [comment]
@MasterOfTheTiger

**Comment by MasterOfTheTiger** [comment]
That would be cool. I don't have much time right now, but when I get back to the project (which I hope can be soon) I will certainly consider this!

**Comment by VFDan** [comment]
> 
> 
> That would be cool. I don't have much time right now, but when I get back to the project (which I hope can be soon) I will certainly consider this!

I can help you with that if you want @MasterOfTheTiger. I was the one who made the dark theme after all.

In fact, I'll fork your project and add the dark theme.

**Comment by locness3** [comment]
Well; shouldn't we close this ?

**Comment by moeenio** [comment]
Well; shouldn't we close this ?

**Comment by MasterOfTheTiger** [comment]
No. It is not fully implemented yet. There is no way to enable it.

**Comment by moeenio** [comment]
Ok, sorry

**Comment by VFDan** [comment]
I'm sorry but I won't be able to implement the rest.

**Comment by MasterOfTheTiger** [comment]
Okay. I may be able to. But don't expect it in a timely manner. Sorry.

**Comment by VFDan** [comment]
I'm almost finished with creating a dark theme, but it's gonna be a userscript.

**Comment by VFDan** [comment]
https://github.com/VFDan/Scratch-Dark-Theme - There are a few minor bugs but I think they can be worked out.

**Comment by MasterOfTheTiger** [comment]
That looks great! I will try it when I get the chance.
